.env.test

```diff
@@ -7,5 +7,7 @@ SECRET_KEY=secret-key
 TRELLO_API_KEY=sdufbsjgldsfb237yrb4r
 TRELLO_API_TOKEN=bdjkb43jkbt3ithh2cbi2hrxiunh1i2h39hi
 TRELLO_BOARD_ID=620e338747c98f4449c740c7
+MONGO_CONNECTION_STRING=mongodb://fakemongo.com
+MONGO_DATABASE_NAME=kpcosmosgroup
```

Comment:
Not vital but it's nice to both:
* Clear out the unnecessary Trello variables
* And match any changes here in your `.env.template` file to make it easier for a new dev to not miss anything